### DISCUSSION TITLE
Grafana alerts firing for KubeControllerManager, KubeScheduler

### DISCUSSION THREAD

**Discussion by h-pcode** [blackcowmoo/Grafana-Google-Analytics-DataSource#71] [General]
Hi I have installed grafana 9.0.5 using prometheus operator and I have been observing 2 alerts firing in grafana that is

1: KubeControllerManagerDown
KubeControllerManager has disappeared from Prometheus target discovery.
2: KubeSchedulerDown
KubeScheduler has disappeared from Prometheus target discovery.

and I have followed some solution like changing the binding address of kube system config from 127.0.0.1:10249 to 0.0.0.0:10249 and others but there is no luck and I am constantly getting that alerts.
Can you please help me on this.

**Comment by lcc3108** [comment]
You can only help us with the Google Analytics grafana plugin here.

I suggest you post your questions on the Prometheus community channel.
